### ISSUE TITLE
训练数据集出现问题

### ISSUE THREAD

**Issue by leehaining** [MhLiao/MaskTextSpotterV3#68]
您好，我在训练icdar2015数据集和自己数据集时都出现了以下问题
`2021-10-28 14:39:23,165 maskrcnn_benchmark.trainer INFO: Start training
2021-10-28 14:39:26,805 maskrcnn_benchmark.trainer INFO: eta: 5:03:07  iter: 0  loss: 12.6127 (12.6127)  loss_classifier: 0.4748 (0.4748)  loss_box_reg: 0.0052 (0.0052)  loss_mask: 5.0979 (5.0979)  loss_char_mask: 0.0000 (0.0000)  loss_seq: 6.0541 (6.0541)  loss_seg: 0.9807 (0.9807)  time: 3.6375 (3.6375)  data: 1.3635 (1.3635)  lr: 0.002036  max mem: 1099
2021-10-28 14:39:44,816 maskrcnn_benchmark.trainer INFO: eta: 1:25:33  iter: 20  loss: nan (nan)  loss_classifier: nan (nan)  loss_box_reg: nan (nan)  loss_mask: 5.1765 (356.7433)  loss_char_mask: 0.0000 (0.0000)  loss_seq: 3.6069 (4.1097)  loss_seg: nan (nan)  time: 0.6030 (1.0309)  data: 0.0082 (0.0800)  lr: 0.002756  max mem: 1906`
loss值在训练过程中变为'**nan**'，暂时还未找到出现这种情况的原因，还请解答，谢谢。
训练命令：`python -m torch.distributed.launch --nproc_per_node 1 tools/train_net.py --config-file configs/pretrain/seg_rec_poly_fuse_feature.yaml`
这是训练中用的seg_rec_poly_fuse_feature.yaml，仅修改了SOLVER部分
![2021-10-29 10-06-06 的屏幕截图](https://user-images.githubusercontent.com/65746712/139362140-290bc9ce-4152-49e7-9b32-5a00007ccaf2.png)
![2021-10-29 10-05-58 的屏幕截图](https://user-images.githubusercontent.com/65746712/139362145-66bd8bf0-6f2b-4026-b17a-665fc113035d.png)
